styleguide/source/_patterns/03-templates/04-topic/topic.twig

```diff
@@ -5,13 +5,9 @@
 
 <main class="topic container">
 
-  <div class="grid grid-margin">
```

Comment:
Why is the title being removed?

Reply:
the title section has been restored. good catch